1.9/administration/upgrading.md

```diff
@@ -19,7 +19,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 - Mesos, Mesos Frameworks, Marathon, Docker and all running tasks in the cluster should be stable and in a known healthy state.
 - For Mesos compatibility reasons, we recommend upgrading any running Marathon-on-Marathon instances to version Marathon 0.16.0-RC3 before proceeding with this DC/OS upgrade.
```

Comment:
@spahl or @aquamatthias do we need to update this Marathon-on-Marathon upgrade requirement for DC/OS 1.9?